.codecov.yml

```diff
@@ -0,0 +1,34 @@
+codecov:
+  require_ci_to_pass: yes
+
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: yes
+  require_base: yes
+  require_head: yes
+
+coverage:
+  range: 70..100
+  status:
+    project:
+      default:
+        target: 70%
```

Comment:
What is the difference between project target and patch target? Is the `patch` only for modifying files and `project` for new files or all files?

Reply:
I just got inspired by the config from iceoryx. I'll look it up later and tell you.

Reply:
Project is for the average of all files in the project and patch for the new code from the PR.